bahmni-docker/docker-compose.yml

```diff
@@ -271,6 +273,68 @@ services:
     profiles: ["default","appointments"]
     # volumes:
     # - "${APPOINTMENTS_PATH:?}/dist/:/usr/local/apache2/htdocs/appointments"
+
+  dcm4chee:
+    image: bahmni/dcm4chee:${DCM4CHEE_IMAGE_TAG:?}
+    profiles: [ "default","pacs" ]
+    environment:
+      DB_HOST: ${PACS_DB_HOST:?}
+      DB_PORT: ${PACS_DB_PORT:?}
+      DB_NAME: ${DCM4CHEE_DB_NAME:?}
+      DB_USERNAME: ${DCM4CHEE_DB_USERNAME:?}
+      DB_PASSWORD: ${DCM4CHEE_DB_PASSWORD:?}
+    ports:
+      - '8055:8055'
+      - '11112:11112'
+    volumes:
+      - 'dcm4chee-archive:/var/lib/bahmni/dcm4chee/server/default/archive'
+    logging: *log-config
+
+  pacs-integration:
+    image: bahmni/pacs-integration:${PACS_INTEGRATION_IMAGE_TAG:?}
+    profiles: [ "default","pacs" ]
+    environment:
+      DB_HOST: ${PACS_DB_HOST:?}
+      DB_PORT: ${PACS_DB_PORT:?}
+      DB_NAME: ${PACS_INTEGRATION_DB_NAME:?}
+      DB_USERNAME: ${PACS_INTEGRATION_DB_USERNAME:?}
+      DB_PASSWORD: ${PACS_INTEGRATION_DB_PASSWORD:?}
+      OPENMRS_HOST: ${OPENMRS_HOST:?}
+      OPENMRS_PORT: ${OPENMRS_PORT:?}
+      OPENMRS_ATOMFEED_USER: ${OPENMRS_ATOMFEED_USER:?}
+      OPENMRS_ATOMFEED_PASSWORD: ${OPENMRS_ATOMFEED_PASSWORD:?}
+    logging: *log-config
+
+  pacsdb:
+    image: postgres:9.6
+    profiles: [ "default","pacs" ]
+    environment:
+      POSTGRES_PASSWORD: ${PACS_DB_ROOT_PASSWORD}
+      DCM4CHEE_DB_NAME: ${DCM4CHEE_DB_NAME:?}
+      DCM4CHEE_DB_USERNAME: ${DCM4CHEE_DB_USERNAME:?}
+      DCM4CHEE_DB_PASSWORD: ${DCM4CHEE_DB_PASSWORD:?}
+      PACS_INTEGRATION_DB_NAME: ${PACS_INTEGRATION_DB_NAME:?}
```

Comment:
Why does the DB need the connection credentials. Does it use these to create a user in the DB for allowing services to connect?

Reply:
Yes Gurpreet! Since dcm4chee and pacs-integration needs its own seperate db's a single postgres instance would be holding the two databases.